examples/.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
tried to extend CRA eslint, but it doesn't kicks in, and failing to find eslint-plugin-react-perf-rules, which in turn fails to run CRA with unknown rule comments in code.